src/modules/functions.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable  prefer-destructuring */
+/* eslint-disable  no-restricted-globals */
```

Comment:
[OPTIONAL] Rather than disable the linters, strive to fix them as this makes you more accustomed to the style guide of the tech.